src/Main.elm

```diff
@@ -83,21 +119,48 @@ buttonToAction button =
 saveAction : Model -> CalButton -> Model
 saveAction model button =
     case ( model.actionA, model.actionB ) of
```

Comment:
I'm not sure what more you're planning/working on but I'd like to point out that with this change you don't need the second value, `model.actionB` in the case statement. It's always wildcard matched.

Reply:
Your right, thanks.